Modules/Loadable/SubjectHierarchy/Widgets/qSlicerSubjectHierarchyViewContextMenuPlugin.cxx

```diff
@@ -138,13 +147,37 @@ void qSlicerSubjectHierarchyViewContextMenuPluginPrivate::init()
     qSlicerSubjectHierarchyAbstractPlugin::SectionDefault, 0);
   QObject::connect(this->MaximizeViewAction, SIGNAL(triggered()), q, SLOT(maximizeView()));
 
+  this->CenterThreeDViewAction = new QAction(tr("Center 3D view"), q);
```

Comment:
We need to keep the names used in the documentation consistent with the names in the context menu. Currently, the names:
- [Center 3D view](https://slicer.readthedocs.io/en/latest/user_guide/user_interface.html#d-view) in both the documentation and in the menu, but "3D" feels somewhat redundant
- [Reset field of view](https://slicer.readthedocs.io/en/latest/user_guide/user_interface.html#slice-view) in the documentation, "Fit view" in the menu. The action centers the view, resizes it to fit, and may reset the orientation as well, so I like the "Fit view" term.

**What do you think about changing the names everywhere to these?**
- Center view (for 3D)
- Fit view (for slice view)

@pieper @jcfr